src/mixer/plugins/volume_mapping.cxx

```diff
@@ -43,6 +41,7 @@
 #endif /* __UCLIBC__ */
 
 #define MAX_LINEAR_DB_SCALE	24
+#define SND_CTL_TLV_DB_GAIN_MUTE   -9999999
```

Comment:
hmm... it seems you are right (again...). Don't know why I decided that I saw an rounding issue earlier with this setting. I'll reset commits back and remove the _GNU_SOURCE line from the first commit.